cpp/include/cudf/io/parquet.hpp

```diff
@@ -552,24 +568,22 @@ class parquet_writer_options {
   /**
    * @brief Sets the maximum row group size, in bytes.
    */
-  void set_row_group_size_bytes(size_t size_bytes)
-  {
-    CUDF_EXPECTS(
-      size_bytes >= 512 * 1024,
-      "The maximum row group size cannot be smaller than the page size, which is 512KB.");
-    _row_group_size_bytes = size_bytes;
-  }
+  void set_row_group_size_bytes(size_t size_bytes) { _row_group_size_bytes = size_bytes; }
 
   /**
    * @brief Sets the maximum row group size, in rows.
    */
-  void set_row_group_size_rows(size_type size_rows)
-  {
-    CUDF_EXPECTS(
-      size_rows >= 5000,
-      "The maximum row group size cannot be smaller than the page size, which is 5000 rows.");
-    _row_group_size_rows = size_rows;
-  }
+  void set_row_group_size_rows(size_type size_rows) { _row_group_size_rows = size_rows; }
```

Comment:
Should this validate the maximum row group size against the maximum page size?
```suggestion
  void set_row_group_size_rows(size_type size_rows)
  {
    CUDF_EXPECTS(
      size_rows >= _target_page_size_rows,
      "The maximum row group size in rows cannot be smaller than the maximum page size in rows.");
    _row_group_size_rows = size_rows;
  }
```